experimental/datacommons-js/example.html

```diff
@@ -19,129 +8,15 @@
-          apiRoot: datacommons.root,
-          isDataTile: false,
-          title:
-            "Population Below Poverty Level Status in Past Year in United States (2020)",
-          place: {
-            dcid: "country/USA",
-            name: "United States of America",
-            types: ["Country"],
-          },
-          statVarSpec: [
-            {
-              name: "Population Below Poverty Level Status in Past Year",
-              statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
-            },
-          ],
-        });
-
-        // Draw map chart
-        datacommons.drawMap(document.getElementById("map-example"), {
-          id: "map-chart-1",
-          svgChartHeight: 200,
-          className: "my-chart",
-          apiRoot: datacommons.root,
-          isDataTile: false,
-          title:
-            "Population Below Poverty Level Status in Past Year in States of United States (2020)",
-          place: {
-            dcid: "country/USA",
-            name: "United States of America",
-            types: ["Country"],
-          },
-          enclosedPlaceType: "State",
-          statVarSpec: {
-            name: "Population Below Poverty Level Status in Past Year",
-            statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
-          },
-        });
-
-        // Draw ranking chart
-        datacommons.drawRanking(document.getElementById("ranking-example"), {
-          placeName: "USA",
-          placeType: "State",
-          withinPlace: "country/USA",
-          statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
-          isPerCapita: false,
-          scaling: 1,
-          unit: "",
-          date: "2020",
-        });
-      };
-    </script>
   </head>
   <body>
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
-      <div class="container">
-        <a class="navbar-brand" href="#">Datacommons JS Embedded Charts</a>
-      </div>
-    </nav>
-    <div class="container">
-      <!-- Web component example -->
-      <h2>Map Web Component</h2>
-      <datacommons-map
-        title="Population Below Poverty Level Status in Past Year in States of United States (2020)"
-        placeDcid="country/USA"
-        enclosedPlaceType="State"
-        statVarName="Population Below Poverty Level Status in Past Year"
-        statVarDcid="Count_Person_BelowPovertyLevelInThePast12Months"
-      ></datacommons-map>
-      <!-- Map chart-->
-      <h2 class="mt-4 mb-4">JS Map Chart</h2>
-      <div id="map-example"></div>
-      <!-- Bar chart-->
-      <h2 class="mt-4 mb-4">JS Bar Chart</h2>
-      <div id="bar-example"></div>
-
-      <!-- Line chart-->
-      <h2 class="mt-4 mb-4">JS Line Chart</h2>
-      <div id="line-example"></div>
-
-      <!-- Ranking chart-->
-      <h2 class="mt-4 mb-4">JS Ranking Chart</h2>
-      <div id="ranking-example"></div>
-    </div>
+    <h1>Datacommons Web Components</h1>
+    <datacommons-map
+      title="Count_Person"
```

Comment:
nit: give it a nicer title like "population"

Reply:
done!